### PR TITLE
feat: Auto extract model when upload model in the UI

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @gaocegege @FogDong 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by gaocegege — COMMENTED** [review]
Can you please add an e2e test?

**Comment by simon-cj** [comment]
> Can you please add an e2e test?

OK

File: testutil/integration/registry_test.go [hunk 1]
```diff
@@ -48,6 +50,13 @@ var _ = Describe("Model Registry", func() {
 				WithMultipart().
 				WithFile("file", "./models/model.zip").WithFormField("model", string(bytes)).
 				Expect().Status(http.StatusCreated)
+			// Upload model will create ModelJob automatically, now get ModelJobList.
+			respStr := e.GET("/api/v1alpha1/namespaces/{namespace}/modeljobs/",
+				"default").Expect().Body()
```
Comment (gaocegege) [inline testutil/integration/registry_test.go:55]: Can we use .JSON() here?

**Pull request merged**